### DISCUSSION TITLE
Send Request with Bearer Token.

### DISCUSSION THREAD

**Discussion by nerisonpitogo** [dedoc/scramble#507] [General]
How do you send a request with Bearer Token? I read the docs and find it hard to figure out.
I can easily make request for endpoints that does not have any authentication. 

![image](https://github.com/user-attachments/assets/d4fda4ff-ce6f-44c0-b233-9f0a421b46e1)

However for endpoint with a bearer token, I have no Idea how to send request in scramble. This is my example in postman.

![image](https://github.com/user-attachments/assets/0202f79b-a3e1-461f-9845-092a6604e865)

and I don't know how to do it in scramble. There is no field on how to attach the token in header.

**Comment by romalytvynenko** [comment]
@nerisonpitogo There is Token input on your first screenshot. Have you tried it?

**Comment by nerisonpitogo** [comment]
OMG I did not notice. Thank you so much.
